src/atoms/cc-input-number.js

```diff
@@ -21,6 +22,9 @@ const decrementSvg = new URL('../assets/decrement.svg', import.meta.url).href;
  * @event {CustomEvent} cc-input-number:requestimplicitsubmit - Fires when enter key is pressed.
  *
  * @cssprop {Align} --cc-input-number-align - Change the alignment of the number present in the input (defaults: `right`).
+ *
+ * @slot help - The help message to be displayed right below the `<input>` element. Please use a `<p>` tag.
```

Comment:
does the "Please use a `<p>` tag" read as a little bit passive aggressive or is it ok ? :sweat_smile:

Reply:
Nope, it's fine :+1: